lib/sensitive_data_filter/middleware/env_filter.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+require 'facets/kernel/present'
+
+module SensitiveDataFilter
+  module Middleware
+    class EnvFilter
+      attr_reader :occurrence
+
+      def initialize(env)
+        @original_env_parser = EnvParser.new(env)
+        @filtered_env_parser = @original_env_parser.copy
+        @scanner             = ParameterScanner.new(@original_env_parser)
```

Comment:
Because this is followed by a function call and then a variable assignment this alignment of equals signs puts me off. Very minor but it's a thing.